### PR TITLE
range-export check-ins script

### PR DESCRIPTION
**Pull request by dpup**
#### What's this PR do?

Initial command for exporting check-ins.

#### Definition of Done:

- [ ] Code is easy to understand and conforms with Prettier & eslint configs
- [ ] Incomplete code is marked with TODOs
- [ ] Code is suitably instrumented with logging and metrics
- [ ] Documentation has been updated as appropriate
- [ ] Manifest has been updated and version incremented correctly
- [ ] [OWASP Top 10](https://www.owasp.org/index.php/Top_10-2017_Top_10) have been considered

### CODE REVIEW

File: cmd.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    const client = new Range();
+    const cmd = commands[cli.input[0]];
+    if (!cmd) fatal(`Unknown command\n${cli.input[0]}`);
+
+    const data = await cmd(client, cli.flags);
+    let fileData;
+    if (cli.flags.fmt === 'csv') {
+      fileData = await json2csvAsync(data, {
+        unwindArrays: true,
+        expandArrayObjects: true,
```
Comment (ghost) [inline cmd.js:110]: Neat. This mimics SQL join output, right?

File: cmd.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+import Range from 'range-sdk';
+import { join, resolve } from 'path';
+import { writeFileSync, statSync } from 'fs';
+
+import pkg from 'json-2-csv';
+const { json2csvAsync } = pkg;
+
+import { createRequire } from 'module';
+const meow = createRequire(import.meta.url)('meow');
```
Comment (ghost) [inline cmd.js:11]: Never seen this syntax before. I see the docs recommend it though.

**Review by ghost — APPROVED** [review]
Looks good. I could see supporting output to stdout eventually (bar and logs going to stderr).

File: cmd.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  `
+  Usage
+    $ range-export <options> <cmd>
+
+  Commands
+    check-ins    Export check-in data
+
+  Options
+    --after, -a   Return data after this ISO timestamp
+    --before, -b  Return data before this ISO timestamp
```
Comment (ghost) [inline cmd.js:30]: What's your vision for branching this out as we add more commands? Eg, not all options will map to all commands.

Comment (dpup) [reply cmd.js:30]: Hadn't thought that far ahead. I've seen Options be a union of all args. I've also seen `range-export check-ins --help`, but not sure `meow` works with that.

**Pull request merged**